Makefile

```diff
@@ -75,7 +75,7 @@ GORELEASER     := bin/goreleaser/v.0.147.2/$(OS)/goreleaser
 GOFUMPT        := $(GOBIN)/gofumpt
 
 $(GOLANGCILINT):
-	$(GO) get github.com/golangci/golangci-lint/cmd/golangci-lint
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/b90551cdf9c6214075f2a40d1b5595c6b41ffff0/install.sh | sh -s -- -b ${GOBIN} v1.32.2
```

Comment:
It's a bit unclear what b90551cdf9c6214075f2a40d1b5595c6b41ffff0 points to. PR title says 1.32.2, shouldb't this url be `https://raw.githubusercontent.com/golangci/golangci-lint/v1.32.2/install.sh` then?

For both cases: Let's imagine somebody wants to update this version or wants to find out where this url comes from, it  would be nice with a link to where you found this url.

Reply:
@yngvark the reason for that line of code is in the commit message. I changed out the branch/release name to the actual commit since we're actually downloading and executing code here. Code in branches can change, release tags can change, but a commit reference can not change. Am I wrong?

Reply:
Good point! Defintely let it stay.

However, could we still add a comment so it's easy to update this commit hash?

Reply:
I'll add a comment describing where I got it from.

However that install script hasn't changed in the last 7 months, so its completely isolated from releases. To bump golangci-lint all we have to do is to change the last word on the line to the correct version